### PR TITLE
Add sub protocol information of Graphql subscriptions to 3.2.0 doc.

### PR DESCRIPTION
**Pull request by dushaniw**

### CODE REVIEW

File: en/docs/learn/design-api/create-api/create-a-graphql-api.md [hunk 1]
```diff
@@ -6,8 +6,13 @@ You can use a Schema Definition Language (SDL) schema to design a GraphQL API in
 
 All GraphQL schemas generally have three operation root types, which are namely query, mutation, and subscription. Therefore, every request against a GraphQL endpoint of a GraphQL server should have a payload starting with any one of the root types, including its related operation name. You can manage the security, authorization, and rate limiting aspect of each operation based on its operation name. 
 
-!!! note
-    GraphQL client supports only `graphql-transport-ws` protocol but the Gateway supports both the `graphql-transport-ws` and `graphql-ws` protocols.
+!!! Important
+    **GraphQL Subscription Operations Support Over Websockets** has been introduced via an U2/WUM update and is effective from 26th February 2022 (2022-02-26).
```
Comment (samgnaniah) [inline en/docs/learn/design-api/create-api/create-a-graphql-api.md:10]: Let's mention the update level as well.

File: en/docs/learn/design-api/create-api/create-a-graphql-api.md [hunk 2]
```diff
@@ -6,8 +6,13 @@ You can use a Schema Definition Language (SDL) schema to design a GraphQL API in
...
+
+    For more information on how to update using U2, see [Updates 2.0 Documentation](https://updates.docs.wso2.com/en/latest/updates/overview/). For more information on how to update using WUM, see the documentation [Using WSO2 Update Manager](https://docs.wso2.com/display/updates100/Using+WSO2+Update+Manager).
+
+    The developer portal's GraphiQL client in the update2/wum updated distribution of API-M 3.2.0 efffective from 26th February 2022 (2022-02-26), supports only [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md) sub protocol.
```
Comment (samgnaniah) [inline en/docs/learn/design-api/create-api/create-a-graphql-api.md:14]:
```suggestion
    The Developer Portal's GraphQL client in the Update 2/WUM updated distribution of API-M 3.2.0 effective from 26th February 2022 (2022-02-26), supports only [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md) sub protocol.
```

File: en/docs/learn/tutorials/create-and-publish-a-graphql-api.md [hunk 3]
```diff
@@ -300,11 +300,14 @@ Click **LIFECYCLE** to navigate to the API lifecycle and click **PUBLISH** to pu
 
 #### Step 5.2 - Optionally, try out a Subscription operation
 
-!!! warning
-    **GraphQL Subscription Operations Support** has been introduced via an U2/WUM update and is effective from 26th February 2022 (2022-02-26).
+!!! Important
+    **GraphQL Subscription Operations Support Over Websockets** has been introduced via an U2/WUM update and is effective from 26th February 2022 (2022-02-26).
```
Comment (samgnaniah) [inline en/docs/learn/tutorials/create-and-publish-a-graphql-api.md:304]: Same comment as above.

File: en/docs/learn/tutorials/create-and-publish-a-graphql-api.md [hunk 4]
```diff
@@ -300,11 +300,14 @@ Click **LIFECYCLE** to navigate to the API lifecycle and click **PUBLISH** to pu
...
 
     For more information on how to update using U2, see [Updates 2.0 Documentation](https://updates.docs.wso2.com/en/latest/updates/overview/). For more information on how to update using WUM, see the documentation [Using WSO2 Update Manager](https://docs.wso2.com/display/updates100/Using+WSO2+Update+Manager).
 
+    The developer portal's GraphiQL client in the update2/wum updated distribution of API-M 3.2.0 efffective from 26th February 2022 (2022-02-26), supports only [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md) sub protocol.
```
Comment (samgnaniah) [inline en/docs/learn/tutorials/create-and-publish-a-graphql-api.md:308]:
```suggestion
    The Developer Portal's GraphQL client in the Update 2/WUM updated distribution of API-M 3.2.0 effective from 26th February 2022 (2022-02-26), supports only [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md) sub protocol.
```

**Review by samgnaniah — APPROVED** [review]

File: en/docs/learn/design-api/create-api/create-a-graphql-api.md (hunk 1 above)
Comment (dushaniw) [reply en/docs/learn/design-api/create-api/create-a-graphql-api.md]: done

**Pull request merged**